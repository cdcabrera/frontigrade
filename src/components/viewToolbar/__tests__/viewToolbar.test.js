import React from 'react';
import { mount } from 'enzyme';
import ViewToolbar from '../viewToolbar';

describe('ViewToolbar Component', () => {
  it('should render', () => {
    const props = {};

    const component = mount(<ViewToolbar {...props} />);

    expect(component.render()).toMatchSnapshot();
  });
});
