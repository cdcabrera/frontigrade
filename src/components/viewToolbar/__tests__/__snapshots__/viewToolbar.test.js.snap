// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewToolbar Component should render 1`] = `
<div
  class="container-fluid"
>
  <div
    class="toolbar-pf row"
  >
    <div
      class="col-sm-12"
    >
      <form
        class="toolbar-pf-actions"
      >
        <div
          class="form-group"
        />
        <div
          class="form-group"
        >
          <button
            class="btn btn-link"
            disabled=""
            type="button"
          >
            <span
              aria-hidden="true"
              class="fa fa-sort-alpha-asc sort-direction"
            />
          </button>
        </div>
        <div
          class="toolbar-pf-action-right"
        >
          <div
            class="form-group"
          >
            <button
              class="btn btn-primary"
              disabled=""
              type="button"
            >
              Add Account
            </button>
            <button
              class="btn btn-default"
              disabled=""
              type="button"
            >
              Export
            </button>
          </div>
        </div>
      </form>
      <div
        class="toolbar-pf-results row"
      >
        <div
          class="col-sm-12"
        >
          <p
            class="filter-pf-active-label"
          >
            No Filters
          </p>
          <h5
            class="cloudmeter-view-count"
          >
            0 Results
          </h5>
        </div>
      </div>
    </div>
  </div>
</div>
`;
